src/test/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineViewTest/testMyUserIdCauseConversionNoStatic/jobs/static-job/builds/2014-11-06_11-46-33/build.xml

```diff
@@ -0,0 +1,200 @@
+                <permission>com.cloudbees.plugins.credentials.CredentialsProvider.Delete:user@user.com</permission>
+                <permission>com.cloudbees.plugins.credentials.CredentialsProvider.View:user@user.com</permission>
+                <permission>hudson.model.Computer.Build:user@user.com</permission>
+                <permission>hudson.model.Computer.Configure:user@user.com</permission>
+                <permission>hudson.model.Computer.Connect:user@user.com</permission>
+                <permission>hudson.model.Computer.Create:user@user.com</permission>
+                <permission>hudson.model.Computer.Delete:user@user.com</permission>
+                <permission>hudson.model.Computer.Disconnect:user@user.com</permission>
+                <permission>hudson.model.Hudson.Administer:user@user.com</permission>
+                <permission>hudson.model.Hudson.ConfigureUpdateCenter:user@user.com</permission>
+                <permission>hudson.model.Hudson.Read:user@user.com</permission>
+                <permission>hudson.model.Hudson.RunScripts:user@user.com</permission>
+                <permission>hudson.model.Hudson.UploadPlugins:user@user.com</permission>
+                <permission>hudson.model.Item.Build:user@user.com</permission>
+                <permission>hudson.model.Item.Cancel:user@user.com</permission>
+                <permission>hudson.model.Item.Configure:user@user.com</permission>
+                <permission>hudson.model.Item.Create:user@user.com</permission>
+                <permission>hudson.model.Item.Create:user@user.com</permission>
+                <permission>hudson.model.Item.Delete:user@user.com</permission>
+                <permission>hudson.model.Item.Discover:user@user.com</permission>
+                <permission>hudson.model.Item.Read:user@user.com</permission>
+                <permission>hudson.model.Item.Read:user@user.com</permission>
+                <permission>hudson.model.Item.Workspace:user@user.com</permission>
+                <permission>hudson.model.Run.Delete:user@user.com</permission>
+                <permission>hudson.model.Run.Delete:user@user.com</permission>
+                <permission>hudson.model.Run.Update:user@user.com</permission>
+                <permission>hudson.model.Run.Update:user@user.com</permission>
+                <permission>hudson.model.View.Configure:user@user.com</permission>
+                <permission>hudson.model.View.Create:user@user.com</permission>
+                <permission>hudson.model.View.Delete:user@user.com</permission>
+                <permission>hudson.model.View.Read:user@user.com</permission>
+                <permission>hudson.scm.SCM.Tag:user@user.com</permission>
+                <!--<permission>org.jfrog.hudson.ArtifactoryPlugin.Promote:user@user.com</permission>-->
+                <!--<permission>org.jfrog.hudson.ArtifactoryPlugin.Release:user@user.com</permission>-->
```

Comment:
Uncomment this and you will have broken jenkins that probably fixed by this https://github.com/jenkinsci/matrix-auth-plugin/pull/1